examples/object-detection/README.md

```diff
@@ -0,0 +1,134 @@
+Other similar datasets may be used though you may prefer to download the KITTI dataset from their web site if you wish to replicate the results from this example.
+
+Optionally you may split the dataset into a training set and a (usually much smaller) validation set.
+Doing so is strongly recommended to assess the quality of the neural network.
+
+The data need to be structured in the following way:
+- An image folder contains supported images (.png, .jpg, .jpeg, .bmp, .ppm).
+- A label folder contains .txt files in KITTI format that define the ground truth.
+Note that for each image in the image folder there must be a corresponding text file in the label folder.
+For example if the image folder includes an image named `foo.png` then the label folder needs to include a file named `foo.txt`.
+
+There needs to be one of the above set of directories for each of the training and validation sets.
+
+### Loading the data into DIGITS
+
+To create an Object Detection dataset in DIGITS, [this version of `3.4-dev`](https://github.com/NVIDIA/DIGITS/tree/2613082eec6c207548c99bb69fa5defc9473b1e8), or any more recent version, is required.
+
+On the DIGITS home page, select the `Datasets` tab then click `Image>Object Detection`:
+
+![select dataset](select-object-detection-dataset.png)
+
+On the dataset creation page, specify the paths to the image and label folders for each of the training and validation sets.
+Other fields can be left to their default value.
+Finally, give your dataset a name and click `Create`:
+
+![dataset form](form-object-detection-dataset.png)
+
+After you have created your dataset you may review data properties by visiting the dataset page.
+In the below example there are 5984 images in the training set and 1496 images in the validation set:
+
+![dataset form](dataset-review.png)
+
+## Model creation
+
+### DetectNet
+
+In this example we will use **DetectNet**.
+DetectNet is a GoogLeNet-derived network that is specifically tuned for Object Detection.
+
+In order to train DetectNet, [NV-Caffe](https://github.com/NVIDIA/caffe) version [0.15.1](https://github.com/NVIDIA/caffe/tree/00f4c1db1dd579d31b1a4f953af61022f6b0da7a) or later is required.
+The [model description for DetectNet](https://github.com/NVIDIA/caffe/tree/caffe-0.15/examples/kitti) may be found in the NV-Caffe repository.
+
+Since DetectNet is derived from GoogLeNet it is strongly recommended to use pre-trained weights from an ImageNet-trained GoogLeNet as this will help speed training up significantly.
+A suitable pre-trained GoogLeNet `.caffemodel` may be found on this [page](https://github.com/BVLC/caffe/tree/rc3/models/bvlc_googlenet).
+
+### Training DetectNet in DIGITS
+
+On the DIGITS home page, select the `Models` tab then click `Image>Object Detection`:
+
+![select dataset](select-object-detection-model.png)
+
+On the model creation page:
+- Select the dataset that was created in the previous section.
+- Set the base learning rate to 0.001.
```

Comment:
`0.0001` works best for me. @ssarathy any problem with that? Your suggested value was twice as big: `0.0002`.